.travis.yml

```diff
@@ -2,14 +2,20 @@ language: c
 
 cache:
   directories:
-    - $HOME/openmpi-gcc
-    - $HOME/openmpi-clang
+    - $HOME/local
+    - $HOME/.luarocks
+    - $HOME/.local
```

Comment:
I don't see the use of this cache directory either from the dep script. This can go also?